### ISSUE TITLE
terraform templates for basic networks

### ISSUE THREAD

**Issue by munjeli** [devopracy/devopracy-cloud#7]
services, public, and build subnets with appropriate acls and routes. Also of course, a vpc.

**Comment by jeffogithub** [comment]
This would be done with terraform, right? Jumping off from something like this? https://github.com/hashicorp/terraform/tree/master/examples/aws-two-tier

**Comment by munjeli** [comment]
Yup that's exactly it. I'd like to see a repo called cloud-common and a folder called terraform. Then put copy those examples in it to begin.
